code/modules/cargo/exports/materials.dm

```diff
@@ -15,36 +15,36 @@
 
 	return M.amount
 
-// Materials. Nothing but plasma is really worth selling. Better leave it all to RnD and sell some plasma instead.
+// Materials. Nothing but Diamonds is really worth selling. Better leave it all to RnD.
```

Comment:
Really drastic changes in prices here. Not a huge fan. It was nice to have other materials actually worth something. Plus, selling phoron is supposed to be a main draw. Science never really lacks for materials. If they must be reduced, the numbers should not be so drastic

Reply:
The issue with that is that phoron is far too common to be really pricy for the ongoing round. Miners frequently end up with multiple stacks that earn hilarious sums for the department. If you send 50 sheets now, it will be 1.500 instead of 10.000 credits. Seeing how phoron is clustered and can have really large nodes, that yield a full stack alone, this makes sense to me.
1 tile of phoron drops 5 ores, ore 2.5 sheets. Should still be possible to run for highscores, but instead of measuring them in the cargo account, people should compare them in mining points. I'd like to reintroduce the credit rewards for miners, but the rate there was 100 credits for 1000 points. I'd go a lot lower if we are to reintroduce it at all (they were completly removed at one point).